tests/stub/iteration/scripts/v4x4/tx_pull_1_nested.script

```diff
@@ -12,14 +12,28 @@ S: RECORD ["1_1"]
 {{
     C: PULL {"n": 1, "[qid]": 1}
     S: RECORD ["1_2"]
-       SUCCESS {}
+       SUCCESS {"has_more": true}
     C: RUN "CYPHER NESTED" {} {}
     S: SUCCESS {"fields": ["x"], "qid": 2}
     C: PULL {"n": 1, "[qid]": 2}
     S: RECORD ["2_1"]
        SUCCESS {"has_more": true}
     C: PULL {"n": 1, "[qid]": 2}
     S: RECORD ["2_2"]
+       SUCCESS {}
+    C: PULL {"n": 1, "[qid]": 1}
```

Comment:
qid can't be optional here as qid 2 is the latest query

Reply:
Good catch, thanks.